includes/version.php

```diff
@@ -14,8 +14,8 @@
  */
 
 define('PROJECT_VERSION_NAME', 'Zen Cart');
-define('PROJECT_VERSION_MAJOR', '1');
-define('PROJECT_VERSION_MINOR', '5.8a');
+define('PROJECT_VERSION_MAJOR', '2');
+define('PROJECT_VERSION_MINOR', '0.0');
```

Comment:
Shouldn't this be `0.0-alpha1`?

Reply:
I'd rather not go down that road unless the group insists.  We're trying to move away from letters, and this is one more thing that has to be remembered on release date that has been messed up in the past.

Reply:
That's part of the standardized versioning.  As we move through alpha/beta versions of v2.0.0, I believe that the alpha/beta designation is quite important.